jest.config.js

```diff
@@ -15,37 +21,43 @@ module.exports = {
       statements: 80,
     },
   },
-  transform: {
-    "\\.js$": "babel-jest",
-    "\\.ts$": "ts-jest"
-  },
   collectCoverageFrom: [
-    '**/*/src/**/*.ts',
-    '!**/index.ts',
-    '!**/__integrationtests__/**',
-    '!**/__mocks__/**',
-    '!**/__tests__/**',
-    '!**/lib/**',
-    '!**/test/**',
-    '!**/kilt/*',
-    '!**/types/**/*',
-    '!**/SDKErrors.ts',
-    '!utils/src/json-schema/*',
-    '!testing/**',
-    '!augment-api/**',
-    '!type-definitions/**',
-    '!**/*.chain.ts',
-    '!did/src/Did.chain.ts',
-    '!did/src/Did.rpc.ts',
-    '!did/src/Did.utils.ts',
-    '!utils/src/jsonabc.ts',
-    '!core/src/utils.ts',
+    'packages/*/src/**',
+  ],
+  coveragePathIgnorePatterns: [
+    '/node_modules/',
+    '/lib/',
+    '/tests/',
+    'packages/types/',
+    'packages/augment-api/',
+    'packages/type-definitions/',
+    'packages/utils/src/json-schema/',
+    'packages/core/src/kilt/',
+    'index.ts',
+    'types.ts',
+    '.chain.ts',
+    'SDKErrors.ts',
+    'Did.chain.ts',
+    'Did.rpc.ts',
+    'Did.utils.ts',
+    'jsonabc.ts',
+    'packages/core/src/utils.ts',
   ],
-  resolver: "ts-jest-resolver",
-  rootDir: 'packages',
-  coverageDirectory: 'coverage',
-  moduleDirectories: [
-    "node_modules",
-    "packages/*/src"
-  ]
 }
+
+module.exports = {
+  ...common,
+  testTimeout: 5000,
+  projects: [
+    {
+      ...common,
+      displayName: 'unit',
+      roots: ['<rootDir>/packages'],
+    },
+    {
+      ...common,
+      displayName: 'breaking',
+      roots: ['<rootDir>/tests/breakingChanges'],
+    },
+  ]
+}
```

Comment:
Will the project benefit from an `.editorconfig` file?

Reply:
looks like it is not well supported in VSCode, which most of the team uses (https://asp.net-hacker.rocks/2020/01/13/editorconfig.html), but it can't hurt either I assume. I did format it though?